pandas_gbq/gbq.py

```diff
@@ -551,23 +551,34 @@ def run_query(self, query, **kwargs):
 
     def load_data(
             self, dataframe, dataset_id, table_id, chunksize=None,
-            schema=None):
+            schema=None, progress_bar=True):
         from pandas_gbq import load
 
         total_rows = len(dataframe)
         logger.info("\n\n")
 
         try:
-            for remaining_rows in load.load_chunks(
-                    self.client, dataframe, dataset_id, table_id,
-                    chunksize=chunksize, schema=schema):
+            chunks = load.load_chunks(self.client, dataframe, dataset_id,
+                                      table_id, chunksize=chunksize,
+                                      schema=schema)
+            if progress_bar and self._check_if_tqdm_exists():
+                from tqdm import tqdm
+                chunks = tqdm(chunks)
+            for remaining_rows in chunks:
                 logger.info("\rLoad is {0}% Complete".format(
                     ((total_rows - remaining_rows) * 100) / total_rows))
         except self.http_error as ex:
             self.process_http_error(ex)
 
         logger.info("\n")
 
+    def _check_if_tqdm_exists(self):
+        try:
+            import tqdm  # noqa
```

Comment:
Since this dependency is optional, there's no need to put it in a function. The only reason the `google-cloud-bigquery` imports are in a function is that they are required and it helps to make a better error message. Ideally there should be no errors if tqdm is not installed.

Example of optional import:

https://github.com/GoogleCloudPlatform/google-cloud-python/blob/ecb501c1d8b099af1d804ad1cfd2cb96575faf3e/bigquery/google/cloud/bigquery/table.py#L24-L27